src/main/java/com/epam/ta/reportportal/ws/model/settings/OAuthRegistrationResource.java

```diff
@@ -27,39 +31,60 @@
  *
  * @author Anton Machulski
  */
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class OAuthRegistrationResource implements Serializable {
 	public static final String URL_PATTERN = "^(http://|https://)?(www\\.)?([a-zA-Z0-9-]+)\\.([a-zA-Z0-9-]+\\.)*[a-z]{2,}(/[a-z_-]+)*$";
 
+	@NotNull
+	@NotBlankString
```

Comment:
Why not use javax.validation.constraints.NotBlank instead of these two annotations?

Reply:
fixed